esp/services/ws_esdlconfig/ws_esdlconfigservice.cpp

```diff
@@ -695,6 +695,29 @@ bool CWsESDLConfigEx::onPublishESDLBinding(IEspContext &context, IEspPublishESDL
     return true;
 }
 
+void setContainerizedEspBindingNameByPort(StringBuffer& espPort, StringBuffer& espBindingName)
+{
+    VStringBuffer xpath("services[@class='esp'][@port='%s']", espPort.str());
+    IPropertyTree* process = getGlobalConfigSP()->queryPropTree(xpath);
+    const char* applicationType = process->queryProp("@type");
```

Comment:
Process could be null.  I think the exception below should be if process is null.  Then if applicationType is null it would be a different wording for the exception.

Reply:
pushed fix